process/ip_database.go

```diff
@@ -65,12 +63,16 @@ IPDatabaseOptions contains custom options for the IPDatabase processor.
 		The database is contextually retrieved using an environment variable and lazy loaded on first invocation. Each environment variable should contain the location of the database, which can be either a path on local disk, an HTTP(S) URL, or an AWS S3 URL.
```

Comment:
docs: Is this comment on the environment variables still accurate after this change? (if inaccurate there are a few more environment variable references in this file)

Reply:
good catch, no longer accurate. will clean that up!